### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by reedrolemodel**
Fix link

### CODE REVIEW

**Review by marcoroth — APPROVED** [review]
Thank you @reedrolemodel!

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Using Stimulus Webpack Helpers
```
Comment (Arezoomohammadalizadeh) [inline README.md:3]: Okay

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
-Call webpack's [`require.context`](https://webpack.js.org/api/module-methods/#require-context) helper with the path to the folder containing your Stimulus controllers. Then, pass the resulting context to the `Application#load` method using the `definitionsFromContext` helper:
+Call webpack's [`require.context`](https://webpack.js.org/api/module-methods/#requirecontext) helper with the path to the folder containing your Stimulus controllers. Then, pass the resulting context to the `Application#load` method using the `definitionsFromContext` helper:
```
Comment (Arezoomohammadalizadeh) [inline README.md:5]: Very good

**Review by Arezoomohammadalizadeh — APPROVED** [review]
thankyou

**Pull request merged**